app/build.gradle

```diff
@@ -74,6 +74,7 @@ dependencies {
     implementation "com.android.support:design:${rootConfiguration.supportLibraryVersion}"
     implementation "com.android.support:cardview-v7:${rootConfiguration.supportLibraryVersion}"
     implementation "com.android.support:support-v4:${rootConfiguration.supportLibraryVersion}"
+    implementation "com.google.android.gms:play-services-auth:${rootConfiguration.playServicesVersion}"
```

Comment:
playStoreImplementation

Reply:
@iamareebjamal Need help, chck gitter!

Reply:
what effects does playstoreImplementation and implementation have
and when I changed to playstoreImplementation, the CI test got failed

Reply:
Read about build flavors